### PR TITLE
add dev container

### PR DESCRIPTION
**Pull request by koichirok**
If possible, could you please add a dev container?

Note that I have selected node14 for the dev container since the lockfileVersion is 1.

### CODE REVIEW

**Comment by amitguptagwl** [comment]
value of VARIANT is different on both places.

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	"extensions": [
+		"dbaeumer.vscode-eslint",
+		"alexkrechik.cucumberautocomplete"
+	],
+
+	// Use 'forwardPorts' to make a list of ports inside the container available locally.
+	// "forwardPorts": [],
+
+	// Use 'postCreateCommand' to run commands after the container is created.
+	// "postCreateCommand": "yarn install",
```
Comment (amitguptagwl) [inline .devcontainer/devcontainer.json:26]: I believe we need to set it to "npm install"

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.205.2/containers/javascript-node
+{
+	"name": "Cytorus",
+	"build": {
+		"dockerfile": "Dockerfile",
+		// Update 'VARIANT' to pick a Node version: 16, 14, 12.
+		// Append -bullseye or -buster to pin to an OS version.
+		// Use -bullseye variants on local arm64/Apple Silicon.
+		"args": { "VARIANT": "14-bullseye" }
```
Comment (amitguptagwl) [inline .devcontainer/devcontainer.json:10]: It should be without "-bullseye", right?

**Review by amitguptagwl — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (koichirok) [reply .devcontainer/devcontainer.json]: Either "bullseye" or "buster" is fine (for me), so I changed it to just "16.
For now, the default (no suffix) is "buster".

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (koichirok) [reply .devcontainer/devcontainer.json]: I changed it to `npm install` and enabled postCreateCommand.

By the way, if package-lock.json is always kept up to date, I beleve `npm ci` is the best choice. `npm install` may  update package-lock.json.

Nevertheless, the package-lock.json in the current HEAD of master does not contain rewire, which I've added for dev dependency, so `npm install` seems to be the better choice at the moment :wink:

Comment (amitguptagwl) [reply .devcontainer/devcontainer.json]: I've just updated package-lock.json. If we change to `npm ci`, we'll have to update package-lock.json always for new dependencies separately.

Comment (koichirok) [reply .devcontainer/devcontainer.json]: I've changed the command to `npm ci`

**Pull request merged**